src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -267,15 +257,66 @@ public function get_total_sale_tooltip() {
 	 * @return string a string of html for the tooltip
 	 */
 	public function get_total_order_tooltip() {
+
+		$message = esc_html__( 'Total Ordered counts tickets from orders of any status, including pending and refunded.', 'event-tickets' );
+
+		return tribe( 'tooltip.view' )->render_tooltip( $message );
+	}
+
+	/**
+	 * Get Pending Tooltip per Ticket
+	 *
+	 * @since TBD
+	 *
+	 * @param $ticket_sold object an object of the ticket to get counts
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_pending_by_ticket_tooltip( $ticket_sold ) {
+
+		$incomplete_statuses = (array) tribe( 'tickets.status' )->get_statuses_by_action( 'count_incomplete', $ticket_sold['ticket']->provider_class, null, true );
+		ob_start();
+			echo esc_html__( 'Pending Order Completion refers to anything with the following statuses:', 'event-tickets' ); ?>
+			<ul class="tooltip-list">
+				<li><?php echo implode( '</li><li>', $incomplete_statuses ) ?></li>
+			</ul>
+		<?php
+		$message = ob_get_clean();
+
+		return tribe( 'tooltip.view' )->render_tooltip( $message );
+	}
+
+
+	/**
+	 * Get Availability Tooltip per Ticket
+	 *
+	 * @since TBD
+	 *
+	 * @param $ticket_sold object an object of the ticket to get counts
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_availability_by_ticket_tooltip( $ticket_sold ) {
+
+		$available[ __( 'Inventory', 'event-tickets' ) ] = $ticket_sold['ticket']->inventory();
+		$available[ __( 'Stock', 'event-tickets' ) ] = $ticket_sold['ticket']->stock();
+		$available[ __( 'Capacity', 'event-tickets' ) ] = $ticket_sold['ticket']->capacity();
+
 		ob_start();
 		?>
-		<div class="tribe-tooltip" aria-expanded="false">
-			<span class="dashicons dashicons-info"></span>
-			<div class="down">
-				<?php echo esc_html__( 'Total Ordered counts tickets from orders of any status, including pending and refunded.', 'event-tickets' ); ?><i></i>
-			</div>
-		</div>
+			<div><?php echo esc_html__( 'Current availablity is using', 'event-tickets' ) . ', ' . esc_html( array_search( min( $available ), $available ) . ' - ' . min( $available ) ); ?></div>
```

Comment:
Should we put this into an admin view file?

Reply:
Should we put it there cause it is more complex then the others? If I load it through an admin file and then send it the tooltip class is that creating to much complexity for a simple message display?

Reply:
@bordoni What do you say about this one?

Reply:
From @bordoni:

```
I say any HTML should be on a a file

specially the ones we want to use `ob_` methods

We should be requiring the use of `Tribe__Template`

more and more where we can

because it gives a ton of filtering for free
```